_includes/header.html

```diff
@@ -2,7 +2,13 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title | escape }}</a>
+    <a class="site-title" href="{{ site.baseurl }}/">
+      {% if site.logo %}
+        <img src="/assets/{{ site.logo }}" class="site-logo" width="150" height="auto">
```

Comment:
How does using JekyllAssets plugin work? Will it create conflicts with this change?